app/common/services/endpoints/post-endpoint.js

```diff
@@ -22,58 +22,24 @@ function (
             params: {
                 order: 'desc',
                 orderby: 'post_date'
-            },
-            transformResponse: [
-                $http.defaults.transformResponse[0],
-                (data) => {
-                    if (!data) {
-                        return data;
-                    }
-
-                    data.results = data.results.map(normalizePost);
-
-                    return data;
-                }
-            ]
+            }
```

Comment:
Accidentally reverted changes??

Reply:
that was on purpose. When I wasn't getting the correct thing back from the DB, I made some unnecessary changes.